package.json

```diff
@@ -59,7 +59,7 @@
     "@types/global-modules": "^2.0.0",
     "@types/html-minifier-terser": "6.1.0",
     "@types/is-ci": "3.0.0",
-    "@types/jest": "26.0.24",
+    "@types/jest": "^29.0.0",
```

Comment:
Setting Jest version range to allow minor updates so that we stay up to date until we come across any issues